Fix inconsistent compiler flags: the problem was that only part of the flags
from makepkg.conf were passed, but _D_FORTIFY_SOURCE flag requires optimization
flags. This broke detection of OpenMP.

diff -r -C3 a/m4/cadna_check_openmp4.m4 b/m4/cadna_check_openmp4.m4
*** a/m4/cadna_check_openmp4.m4	2019-12-16 10:58:25.000000000 -0500
--- b/m4/cadna_check_openmp4.m4	2020-02-28 20:42:40.312822803 -0500
***************
*** 6,12 ****
  
  CADNA_CHECK_OPENMP4_CXXFLAGS="$CXXFLAGS"
  
! CXXFLAGS="$AM_CXXFLAGS $OPENMP_CXXFLAGS   -Werror $OPENMP_LDFLAGS "
  
  
  
--- 6,12 ----
  
  CADNA_CHECK_OPENMP4_CXXFLAGS="$CXXFLAGS"
  
! CXXFLAGS="$CXXFLAGS $AM_CXXFLAGS $OPENMP_CXXFLAGS   -Werror $OPENMP_LDFLAGS "
  
  
  
diff -r -C3 a/m4/cadna_check_openmp_flags.m4 b/m4/cadna_check_openmp_flags.m4
*** a/m4/cadna_check_openmp_flags.m4	2019-12-16 10:58:25.000000000 -0500
--- b/m4/cadna_check_openmp_flags.m4	2020-02-28 20:42:40.312822803 -0500
***************
*** 5,11 ****
  
  CADNA_CHECK_OPENMP_CXXFLAGS="$CXXFLAGS"
  
! CXXFLAGS="$AM_CXXFLAGS $OPENMP_CXXFLAGS  -Werror "
  
  
  
--- 5,11 ----
  
  CADNA_CHECK_OPENMP_CXXFLAGS="$CXXFLAGS"
  
! CXXFLAGS="$CXXFLAGS $AM_CXXFLAGS $OPENMP_CXXFLAGS  -Werror "
  
  
  
